test/server/cmd/grpc_server.go

```diff
@@ -16,16 +17,16 @@ var grpcServerLog = grpcLog.WithName("server")
 
 type grpcServer struct {
 	api.UnimplementedGreeterServer
-	api.UnimplementedHealthServer
+	grpchealth.UnimplementedHealthServer
```

Comment:
Should this just be a different PR?

Reply:
I had it put it here because `kuma-dp` inplicitly use this pb and it crashes and complained that the same proto was found from two packages.

Reply:
I understand this. My point was just that this should be a different PR like: https://github.com/kumahq/kuma/pull/10624#discussion_r1713449313

Reply:
Sorry I can't agree with it. I did not encounter this issue before I made this change and when I ran into the problem, the most straitforward way is to include the fix in as part of this atomic PR.

Reply:
Thanks for the suggestion, I'll improve the practice in future PRs.